src/main/java/org/truffleruby/core/TruffleSystemNodes.java

```diff
@@ -182,7 +182,7 @@ public Object fullMemoryBarrier() {
         public Object log(DynamicObject level, DynamicObject message,
```

Comment:
This Specialization will fail if more than 3 different levels  are passed (already a problem before).
Probably not common as the level is normally a literal Symbol.